components/DataRequirementsViewer.tsx

```diff
@@ -0,0 +1,83 @@
+import { useEffect, useState } from 'react';
+import { useRecoilState } from 'recoil';
+import { selectedMeasureState } from '../atoms/selectedMeasure';
+import { Loader, ScrollArea, Card, Center, Tabs, Title, Text } from '@mantine/core';
+import dynamic from 'next/dynamic';
+const DynamicReactJson = dynamic(import('react-json-view'), { ssr: false });
+
+/**
+ * Pings DEQM-test-server for the $data-requirements results based on the measure
+ * selected. Updates each time a new measure is selected
+ * @returns JSX for Data Requirements panel on screen
+ */
+export default function DataRequirementsViewer() {
+  const [selectedMeasure] = useRecoilState(selectedMeasureState);
+  const [dataRequirements, setDataRequirements] = useState<fhir4.DataRequirement | null>(null);
+  const [error, setError] = useState<fhir4.OperationOutcome | null>(null);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState(false);
```

This is a nitpick, but in Typescript I prefer to omit type decs (in this case a generic), when it's a very simple type that will never change, like a boolean